src/classes/contexts/CommandContext.ts

```diff
@@ -61,23 +61,47 @@ export class CommandContext implements CommandContextBuilder {
 	public async react(emoji: EmojiIdentifierResolvable) {
 		return await this.message.react(emoji);
 	}
+	
+	public async reacts(emojis: EmojiIdentifierResolvable[]) {
+		for (const emoji of emojis) {
+			await this.message.react(emoji);
+		}
+	}
 
-	public async removeReactions() {
+	public async removeReactionsAll() {
 		return this.message.reactions.removeAll();
 	}
```

Comment:
I think that it could be better to just add an overload of `CommandContext#removeReactions` without any arguments, and it would remove all reactions